redfish-exporter/listener.go

```diff
@@ -185,10 +187,16 @@ func processRequest(AppConfig Config, conn net.Conn, req *http.Request, eventCou
 		log.Printf("Message Args: %v", messageArgs)
 		log.Printf("Origin Of Condition: %s", originOfCondition)
 		for _, triggerEvent := range AppConfig.TriggerEvents {
-			if eventType == triggerEvent.EventType && eventId == triggerEvent.EventId && severity == triggerEvent.Severity {
+			redfishServerIP := fmt.Sprintf("https://%v", remote)
+			if eventType == triggerEvent.EventType && eventId == triggerEvent.EventId {
 				log.Printf("Matched Trigger Event: %s with action %s", triggerEvent.EventId, triggerEvent.Action)
-				// TODO: Add the SLURM integration here
 				// Sending event belings to redfish_utils. Each server may have different slurm node associated, and redfish_servers has the info/map.
+				for _, redfishServer := range AppConfig.RedfishServers {
```

Comment:
There’s a duplicated code here:
https://github.com/nod-ai/ADA/blob/main/redfish-exporter/redfish_utils.go#L193
I didn’t expect the SLURM node to be directly mapped to its Redfish server, but we could probably modify the function to something like `getServerInfo` and reuse it.

Reply:
Done.